R/turns.R

```diff
@@ -88,6 +88,10 @@ method(print, Turn) <- function(x, ...) {
   invisible(x)
 }
 
+completion_turn <- function(role = NULL, ...) {
```

Comment:
What do you think about calling this `assistant_turn()` and dropping the `role` argument?

Reply:
That was my first inclination too, but I wasn't sure if there is (or would be) an instance where we'd get a `role` that's something other than `"assistant"`. It seems like a safe assumption (as evidenced by it not being included in some API responses), but then I worried about swinging too far the other way.

Anyway... I think you're right and it should be `assistant_turn()`. Everything downstream is builds on `role = "assistant"` for these turns. I'll make that change.

Reply:
Done in 89582b43c868b05f727b76fd9ac5c6af364a10ce